### ISSUE TITLE
trezor new usb id problem

### ISSUE THREAD

**Issue by molecular** [akhavr/electrum-dash#71]
Hi,

first: thanks for doing electrum-dash, I love it!

I have a problem with trezor. I usually use an older version of trezorlib (0.7.16) because electrum-dash wont recognize more recent versions (it also doesn't work with trezor bridge (trezord) it seems, but I guess thats a different issue).

Now trezor seems to have changed the USB id (both vendor and product) of the trezor in recent firmware upgrade. Hence trezorlib 0.7.16 doesn't recognize trezors with newer firmwares.

Apart from using a trezor with an older firmware or hacking trezorlib 0.7.16 to recognize the new usb id, what else can I do to make this work?

**Comment by akhavr** [comment]
You've not mentioned which version of dash electrum are you using.  I guess you've not updated it for some time, since we're using the following dependencies on trezorlib:

```
3.2.4-3.2.5 -- trezor==0.10.2
3.2.2-3.2.3 -- trezor==0.10.1
3.1.3 -- trezor==0.9.1
3.0.6 -- trezor==0.7.16
```

Check it here https://github.com/akhavr/electrum-dash/blob/master/contrib/deterministic-build/requirements-hw.txt#L106

Upgrade might also fix your #72

**Comment by molecular** [comment]
oh damnit, I'm on master branch, but I just saw that my "git pull" had failed since I made a (bugfix) change.

I'm sorry, my head was 305 commits (10 months) in the past... no wonder those issues.

Thanks for your help!!
